controllers/flowlogspipeline/flp_test.go

```diff
@@ -560,24 +580,21 @@ func TestPipelineTraceStage(t *testing.T) {
 	flp := getFLPConfig()
 
 	b := newBuilder("namespace", corev1.ProtocolUDP, &flp, nil, nil, "", true)
-	stages, parameters := b.buildPipelineConfig()
+	stages, parameters, err := b.buildPipelineConfig()
+	assert.NoError(err)
 	assert.True(validatePipelineConfig(stages, parameters))
 	jsonStages, _ := json.Marshal(stages)
 	assert.Equal(`[{"name":"ipfix"},{"name":"enrich","follows":"ipfix"},{"name":"loki","follows":"enrich"},{"name":"stdout","follows":"enrich"},{"name":"aggregate","follows":"enrich"},{"name":"prometheus","follows":"aggregate"}]`, string(jsonStages))
 }
 
-//go:embed test_metrics_definitions
-var TestFlpMetricsConfig embed.FS
-
 func TestMergeMetricsConfigurationNoIgnore(t *testing.T) {
 	assert := assert.New(t)
 
 	flp := getFLPConfig()
-	FlpMetricsConfigDir = "test_metrics_definitions"
-	FlpMetricsConfig = TestFlpMetricsConfig
 
```

Comment:
The idea here was to use a well-known (unchanging) set of metrics for testing, and not the current version in metrics_definitions, which might change from time-to-time. If we use the regular metrics_definitions, they whenever we make changes to metrics_definitions we may also have to update the test.

Reply:
Ok, but on the other hand, there's also some value in testing the actual embedded metrics (making sure it's correctly formed, etc.). Could be a bit more maintenance , but it also increases coverage